libraries/stdlib/js-ir/runtime/atomicfu.kt

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package kotlin.js
```

Comment:
PTAL: @anton-bannykh

Reply:
Why do these declarations need to be inside stdlib?

Reply:
Atomicfu declarations are no longer in stdlib, I moved them into a separate project `kotlinx-atomicfu-runtime` that is added as a runtime dependency to the plugin.

Reply:
Cool! :+1:

Reply:
(Just a side note) From what I see it should be possible to just "inline" those declarations by the plugin itself, if that's ever needed.